### PR TITLE
Add entries to 2018

### PR DESCRIPTION
**Pull request by juleoc02**

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -1815,5 +1815,517 @@ @InProceedings{MB_paper_B
...
+    title={Interior Point Methods and Preconditioning for PDE-Constrained Optimization Problems Involving Sparsity Terms},
+    author={John W. Pearson and Margherita Porcelli and Martin Stoll},
+    year={2018},
+    eprint={1806.05896},
+    archivePrefix={arXiv},
+    primaryClass={math.OC}
+}
+
+@misc{mohebujjaman2018second,
+    title={Second order ensemble simulation for MHD flow in Elsässer variable with noisy input data},
```
Comment (tjhei) [inline publications-2018.bib:1828]: You need to write \"a

File: publications-2018.bib [hunk 2]
```diff
@@ -1815,5 +1815,517 @@ @InProceedings{MB_paper_B
...
+    author={Hong Zhang and Yunrui Guo and Weibin Li and Paul Andries Zegeling},
+    year={2018},
+    eprint={1801.07182},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+@misc{badia2017fempar,
+    title={FEMPAR: An object-oriented parallel finite element framework},
+    author={Santiago Badia and Alberto F. Martín and Javier Principe},
```
Comment (tjhei) [inline publications-2018.bib:1909]: > Martín
This is also an invalid character

File: publications-2018.bib (hunk 1 above)
Comment (bangerth) [reply publications-2018.bib:1828]: More specifically:
```suggestion
    title={Second order ensemble simulation for MHD flow in Els{\"a}sser variable with noisy input data},
```

File: publications-2018.bib (hunk 2 above)
Comment (bangerth) [reply publications-2018.bib:1909]:
```suggestion
    author={Santiago Badia and Alberto F. Mart{\'i}n and Javier Principe},
```

**Comment by bangerth** [comment]
@juleoc02 -- ping?

**Review by bangerth — APPROVED** [review]

**Pull request merged**